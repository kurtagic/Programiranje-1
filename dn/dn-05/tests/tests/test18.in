94
14 47
14 17
14 25
14 53
14 31
14 78
14 38
14 51
14 68
14 53
14 29
14 96
14 49
14 36
14 82
14 53
14 33
14 85
14 72
14 15
14 20
14 33
14 58
14 67
14 67
14 22
14 14
14 80
14 75
14 88
14 40
14 82
14 59
14 90
14 79
14 70
14 28
14 19
14 79
14 58
14 89
14 24
14 28
14 16
14 70
14 22
14 59
14 50
14 54
14 67
14 42
14 86
14 23
14 27
14 66
14 82
14 24
14 92
14 18
14 35
14 19
14 81
14 24
14 67
14 41
14 77
14 43
14 96
14 52
14 87
14 84
14 39
14 82
14 20
14 39
14 28
14 53
14 91
14 75
14 19
14 78
14 87
14 81
14 50
14 88
14 77
14 94
14 36
14 23
14 21
14 45
14 87
14 20
14 32
