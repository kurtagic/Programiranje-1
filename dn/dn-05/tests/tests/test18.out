14: 6
15: 5
16: 7
17: 6
18: 8
19: 12
20: 16
21: 15
22: 15
23: 13
24: 11
25: 11
26: 9
27: 10
28: 11
29: 10
30: 10
31: 3
32: 7
33: 6
34: 6
35: 6
36: 7
37: 6
38: 6
39: 10
40: 9
41: 9
42: 5
43: 7
44: 6
45: 8
46: 7
47: 7
48: 1
49: 6
50: 12
51: 11
52: 11
53: 11
54: 10
55: 10
56: 7
58: 5
59: 6
60: 5
61: 5
66: 10
67: 11
68: 10
69: 10
70: 6
71: 5
72: 5
73: 1
74: 1
75: 12
76: 11
77: 13
78: 12
79: 12
80: 10
81: 12
82: 11
83: 11
84: 9
85: 7
86: 7
87: 12
88: 11
89: 11
90: 9
91: 6
92: 5
93: 5
94: 6
95: 5
96: 5
97: 3
98: 3
